observability/config.libsonnet

```
@@ -8,6 +8,10 @@ local thanos = (import '../services/observatorium-metrics.libsonnet').thanos;
     targetGroups+:: {
       namespace: 'thanos_status',
     },
+    // Filter the namespaces in thanos-recieve-controller dashboard
+    hierarcies+:: {
```

```suggestion
    hierarchies+:: {
```
But i'm assuming this typo comes from the mixin 🤣

---

Indeed: https://github.com/observatorium/thanos-receive-controller/blob/1a5357d4bd43d4291a0506ecaf09854ecb5ca6e6/jsonnet/thanos-receive-controller-mixin/config.libsonnet#L6

I guess we could fix that first though?

---

Attempted to fix the typo upstream while keeping backwards compatibility, but haven't managed to find a good solution yet. No reason for this to be stuck (imo), so I'll merge this for now.